exercise/views.py

```diff
@@ -362,3 +380,6 @@ def get(self, request, *args, **kwargs):
 
         return HttpResponse(bytedata.decode('utf-8', 'ignore'),
             content_type='text/plain; charset="UTF-8"')
+
+class ExamsStudentView(BaseView):
+    template_name = ""
```

Comment:
Is `ExamsStudentView` unfinished or does it do something useful?

Reply:
It was probably meant to be used in the exams exercises. However, the exam questions use the `ExerciseView` class instead. Using same view class as normal exercises works fine at the moment, but it can have problems in the future. The separate view class would make sense, since they use different templates.

Reply:
We could use `ExamStudentView` as the exam exercise page. It could inherit the normal `ExerciseView`, but override the `template_name` variable since exam pages use different templates than normal chapter pages, but otherwise the logic of viewing and submitting exercises is the same. Then, the `ExerciseView` does not need to use any if statements to select between normal and exam templates.